lib/tasks/achievements_import.rake

```diff
@@ -0,0 +1,14 @@
+namespace :desperado do
+
+  desc 'Import achievements into the DB.'
+  task :achievements_import, [:data_file] => :environment do |_, args|
+    File.open("#{Rails.root}/#{args.data_file}", 'r') do |data|
+      achievements = Psych.load_stream data
+      achievements.each do |achievement|
+        Achievement.create achievement
+      end
```

Comment:
I've got a hunch that ActiveRecord is clever enough to create multiple objects if it gets an array or iterable of hashes.

TryAchievement.create achievements`